bin/cloud-provision.sh

```diff
@@ -16,6 +16,53 @@ fi
 NOVARC_PATH=$1
 SPI_CREDENTIALS_PATH=$2
 OPENSTACK_CREDENTIALS_DIR="$JENKINS_HOME/.openstack"
+PRIVATE_KEY_PATH=~/.canonistack/${OS_USERNAME}_${OS_REGION_NAME}.key
```

Comment:
Ok, added as last argument to the create and provision script, with de default value being `~/.canonistack/${OS_USERNAME}_${OS_REGION_NAME}.key`